File: 268-missing-number/missing-number.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var missingNumber = function(nums) {
    let n=nums.length;
    let sum1=n*(n+1)/2;
    let sum2=0;
    for(let i of  nums){
        sum2+=i
    }
    return sum1-sum2;
};

File: 268-missing-number/missing-number.py
class Solution(object):
    def missingNumber(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        n=len(nums);
        sum=0;
        for el in nums:
            sum+=el;
        sum2=n*(n+1)/2;
        return sum2-sum
        